content/versionedFeedId.ttl

```diff
@@ -0,0 +1,9 @@
+@prefix : <http://glossary.orcfax.io/#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+:versionedFeedId
+    skos:prefLabel "versioned feed id"@en;
+    skos:definition '''
+      It is the extension of the feed id to contain version information.
+      It is constructed as `{{feed-id}}/{{feed-version}}`
```

Comment:
```suggestion
      It is constructed as `{{feed-id}}-{{feed-version}}`
```

Reply:
?!?! Are you sure 
Hyphen is for CER feed names.
 `/` is our (versioned)-feed-id delimiter.

Reply:
Ah, ignore me, you're exactly right.